src/create-reducer.ts

```diff
@@ -92,6 +129,8 @@ export function createReducer<TState, TRootAction extends Action = RootAction>(
         );
       }
       return reducer(state, action);
+    } else if (defaultReducer && !initializationActionTypes.test(action.type)) {
```

Comment:
`!initializationActionTypes.test(action.type)` is not needed user code in default handler should handle the @@redux/init action if needed by just returning the unchanged state.

Reply:
If the return of the defaultHandler is different from initialState, it may not work as intended by the user (due to the redux INIT action triggering the defaultHandler). This might be confusing for some users who are not aware of the existence of redux INIT. What do you think about this issue?

Reply:
That is a trade-off. The default use case is to clear the state and that's that we are aming for to solve as main case. If someone want to use it for some other rare edge case adding that condition is not the issue. We just need to put that info in the docs.

Reply:
@jin60641 could you please add it to the readme?

Reply:
done 33f9db8e1476ac646551c2d5a405b7ed57cd20cc